2.2/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==20.0.4
-django==2.2.12
+django==2.2.13
 psycopg2==2.8.5
 gevent==1.4.0
```

Comment:
Apparently `gevent==1.5.0` came out a few days after our last bump and should work with all the versions of Python we support. Could we bump that while we're at it?

Reply:
I'm aware that `gevent==1.5.0` came out. It broke the builds with what looks like a dependency on a new native library. I tried to satisfy it, but wasn't able to make much progress, and decided to just skip the bump.

I'll create an issue to give it more scrutiny.

Reply:
Ooh, nice catch, agreed on creating an issue. Thanks!